xml/System.Security.Cryptography.Pkcs/Pkcs12Builder.xml

```diff
@@ -58,11 +58,26 @@
         <Parameter Name="pbeParameters" Type="System.Security.Cryptography.PbeParameters" />
       </Parameters>
       <Docs>
-        <param name="safeContents">To be added.</param>
-        <param name="passwordBytes">To be added.</param>
-        <param name="pbeParameters">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="safeContents">The contents to add to the PFX.</param>
+        <param name="passwordBytes">The byte array to use as a password when encrypting the contents.</param>
+        <param name="pbeParameters">The password-based encryption (PBE) parameters to use when encrypting the contents.</param>
+        <summary>Add contents to the PFX in an bundle encrypted with a byte-based password from a byte array.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The password bytes are passed directly into the Key Derivation Function (KDF) used by the algorithm indicated by `pbeParameters`.
+ This enables compatibility with other systems which use a text encoding other than UTF-8 when processing passwords with PBKDF2 (Password-Based Key Derivation Function 2).
+
+ The contents of `safeContents` are read during the call to this method, encrypted, and stored internally as encrypted content.
```

Comment:
Since this is repeated for each of the overloads, you could create a MemberGroup and put it in the Remarks section there.